dlio_benchmark/utils/utility.py

```diff
@@ -51,12 +57,83 @@ def utcnow(format=LOG_TS_FORMAT):
     return datetime.now().strftime(format)
 
 
-def get_rank():
-    return MPI.COMM_WORLD.rank
-
-
-def get_size():
-    return MPI.COMM_WORLD.size
+# After the DLIOMPI singleton has been instantiated, the next call must be
+# either initialize() if in an MPI process, or set_parent_values() if in a
+# non-MPI pytorch read_threads child process.
+class DLIOMPI:
+    __instance = None
+    class MPI_state(Enum):
```

Comment:
Following code convention in the repo, lets use MPIState and you can add it to the enumerations.

Reply:
Are you saying that you want it added to enumerations.py?   Because the enum has no meaning outside of the DLIOMPI class, it is completely internal to the class.

Reply:
I know, for now we don’t have any meaning. But eventually if we implement other communication premitives, its would be better. It’s just a code convention, we have been following.